controllers/topology/internal/extensions/patches/inline/json_patch_generator_test.go

```diff
@@ -735,6 +942,24 @@ func TestRenderValueTemplate(t *testing.T) {
 			},
 			want: &apiextensionsv1.JSON{Raw: []byte(`true`)},
 		},
+		// Default variables via template
```

Comment:
@CecileRobertMichon Turns out it is possible to use variables as default value for variables. It just has to be done by using a variable template in the patch instead of doing it on the variable directly.

Reply:
We should document patches tips and tricks...

Reply:
Definitely. Something like a syntax + cool things to do user-facing doc would be great.